repos/fdbt-site/tests/testData/mockData.ts

```diff
@@ -244,6 +244,7 @@ export const getMockContext = ({
         req,
         pathname: '',
         query,
+        // eslint-disable-next-line react/display-name
```

Comment:
Why is this needed? I tried removing it and running the tests but they all still passed